### PR TITLE
Added help center (user side)

### PR DESCRIPTION
**Pull request by navvyua**
- User can create request in Help center
- User can reply in request
- Added pundit authorization on request pages.
- Request design looks like chat.

Other changes:
- Notifications now use slide animation instead fade.
- Added cool border for messages like _You don't have characters_,
- Changed back location in self created authentication authorization, from profile to root.
- Character method **full_name** moved from model to decorator.
- Added user method **helper_or_admin**?
- _Help ticket_ changed to _Help request_.

### CODE REVIEW

File: app/assets/javascripts/notification.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-  $('.notification').fadeIn('slow');
+  $('.notification').slideDown('slow');
 
   setTimeout(function() {
-    $('.notification').fadeOut('slow', function() {
+    $('.notification').slideUp('slow', function() {
       $(this).remove();
     });
-  }, 4000);
+  }, 6000);
```
Comment (IlyaUmanets) [inline app/assets/javascripts/notification.js]: I think it's very slow

File: app/models/request.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Request < ApplicationRecord
+  belongs_to :user
+  belongs_to :admin, class_name: 'User', required: false
```
Comment (IlyaUmanets) [inline app/models/request.rb]: Order it alphabetically

File: app/models/user.rb [hunk 3]
```diff
@@ -1,7 +1,12 @@
 class User < ApplicationRecord
-  has_many :accounts, dependent: :delete_all
+  has_secure_password
+
   has_many :news
```
Comment (IlyaUmanets) [inline app/models/user.rb]: Order by alpha

File: app/policies/request_policy.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    @user = user
+    @request = request
+  end
+
+  def index?
+    user_owner_or_admin_only
+  end
+
+  def show?
+    user_owner_only || @user == @request.admin
```
Comment (IlyaUmanets) [inline app/policies/request_policy.rb]: `@user == @request.admin` move to method

File: app/views/shared/_sidebar.html.slim [hunk 5]
```diff
@@ -19,8 +19,8 @@ aside.main-sidebar
             span.pull-right-container = fa_icon 'angle-down', class: 'pull-right'
             
           ul.treeview-menu
-            li = link_to fa_icon('plus', text: t('.create_help_ticket'))
-            li = link_to fa_icon('list', text: t('.my_help_tickets'))
+            li = link_to fa_icon('plus', text: t('.create_help_request')), new_user_request_path(current_user)
```
Comment (IlyaUmanets) [inline app/views/shared/_sidebar.html.slim]: Long line

File: app/views/shared/_sidebar.html.slim [hunk 6]
```diff
@@ -19,8 +19,8 @@ aside.main-sidebar
...
+            li = link_to fa_icon('list', text: t('.my_help_requests')), user_requests_path(current_user)
```
Comment (IlyaUmanets) [inline app/views/shared/_sidebar.html.slim]: Same

File: app/assets/javascripts/notification.js (hunk 1 above)
Comment (navvyua) [inline app/assets/javascripts/notification.js]: Changed to 5 seconds.

File: app/models/request.rb (hunk 2 above)
Comment (navvyua) [inline app/models/request.rb]: Ordered.

File: app/models/user.rb (hunk 3 above)
Comment (navvyua) [inline app/models/user.rb]: Ordered.

File: app/policies/request_policy.rb (hunk 4 above)
Comment (navvyua) [inline app/policies/request_policy.rb]: Moved to user_processing_admin_only.

File: app/views/shared/_sidebar.html.slim (hunk 5 above)
Comment (navvyua) [inline app/views/shared/_sidebar.html.slim]: Fixed.

File: app/views/shared/_sidebar.html.slim (hunk 6 above)
Comment (navvyua) [inline app/views/shared/_sidebar.html.slim]: Fixed.

**Pull request merged**